packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -900,6 +908,17 @@ export default class NewVm extends BaseComponent {
   _getRedirectionUrl = id =>
     this.state.state.multipleVms ? '/home' : `/vms/${id}`
 
+  _handleHvmBootFirmware = ({ target: { value } }) => {
+    if (value === 'uefi') {
+      confirm({
```

Comment:
Add a `TODO` to explain that the confirm should be removed once the feature is stabilized.